Vision/README.md

```diff
@@ -11,39 +9,37 @@ Our vision is for a World Wide Web that is more inclusive, and more respectful o
-
-# The Operating Principles of the Consortium
-
-In our operations and membership, we will:
-*	Provide an open forum, where diverse voices from around the world and industries come together, incubate and build consensus for global standards for Web technologies, including organizations of all sizes (from single-person companies to multi-nationals) and considering public feedback.
-* Encourage incubation in new areas and industries with open platforms for discussion, collaboration and innovation, making it more structured and improving consensus-building among key stake holders.
-* Strive for the broadest participation, along axes including worldwide participation, diversity and inclusion, facilitating balance, equity and cooperation among the participants from different industries, user groups and organizational sizes, and thus establishing W3C as representative of the whole community.
-* Increase involvement of under-represented key stakeholders such as end users, content creators, developers etc.
-* Ensure transparency, equity and fairness. Our work will not be exclusively dominated by any person, company or interest group.
-* Establish and improve collaborative relationships with other organizations in the domain of Internet and Web standards, including building and maintaining respected relationships with governments and businesses for providing credible advice.
+# Vision for the W3C
+
+The W3C is an association where diverse voices from around the world and industries come together to incubate and build consensus for the global standards that make up the Web, serving the Vision for the Web.
+
+* We put the needs of users first: above authors, publishers, implementers, paying W3C Members, or theoretical purity.
+* This organization believes in diversity and inclusion of participants from different geographical locations, cultures, languages, accessibility needs, gender identities, and more. 
+* We strongly emphasize accessibility, internationalization, privacy and security.
+* Our standards are rooted in a strong royalty-free patent policy and open copyright licenses, openly developed with consensus of industry and key stakeholders.
+
+# Mission of the W3C
+
+The fundamental purpose of the W3C is to provide an open forum, where diverse voices from around the world and industries come together, incubate and build consensus for global standards for Web technologies, including organizations of all sizes (from single-person companies to multi-nationals) and considering public feedback.
+
+As the W3C leads the Web forward, our mission is to recognize and embody fundamental values and principles into the architecture of the web.  We must become more principled in our execution of the vision of the Web.
+
```

Comment:
This purpose/mission section may still need work.

Reply:
Yup, needs work but is basically OK

Reply:
Yes, the lead sentences in these two sections seem to echo each other